knpu/authorization-code.rst

```diff
@@ -52,14 +50,13 @@ how many eggs each farmer has collected. The only missing piece is OAuth:
 getting the access token for each user so that we can make an API request
 to count their eggs.
 
-Redirecting to Authorize
-------------------------
+## Redirecting to Authorize
 
 Before TopCluck can make an API request to COOP to count my eggs, I need
 to authorize it. On the homepage, there's already an Authorize link, which
 just prints a message right now.
 
-The code behind this URL lives in the ``src/OAuth2Demo/Client/Controllers/CoopOAuthController.php``
+The code behind this URL lives in the `src/OAuth2Demo/Client/Controllers/CoopOAuthController.php`
 file. You don't even need to understand how this works, just know that whatever
 we do here, shows up::
```

Comment:
Ohhh I missed all this codeblocks because there is not a codeblock definition on the file =/